common/docs/docs.jsx

```diff
@@ -39,6 +39,17 @@ export default DocView = React.createClass({
   getMeteorData() {
     const sub = Meteor.subscribe("CacheDocs", this.props.params);
 
+    // If no params have been given, load params from default TOC
+    let params = this.props.params;
+    if (Object.keys(params).length === 0) {
+      let defaultToc = ReDoc.Collections.TOC.findOne({ default: true });
+      if (!!defaultToc) {
+        params.repo = defaultToc.repo;
+        params.branch = defaultToc.branch || Meteor.settings.public.redoc.branch || "master";
+        params.alias = defaultToc.alias;
+      }
+    }
+
```

Comment:
This is being done in the publication around here: https://github.com/reactioncommerce/redoc/blob/master/packages/redoc-core/server/publications.js#L42

Is it necessary to do this on the client as well? 

One of my previous PR's removed this logic because I felt it was duplicated: https://github.com/reactioncommerce/redoc/pull/32/files#diff-695c0e3f85d3e7e81009aa46176472a5L49